frontend/public/components/edit-yaml.jsx

```diff
@@ -50,19 +51,40 @@ const stateToProps = ({ k8s, UI }) => ({
   models: k8s.getIn(['RESOURCES', 'models']),
 });
 
+const getModel = (models, obj) => {
+  if (_.isEmpty(obj) || !models) {
+    return null;
+  }
+  return models.get(referenceFor(obj)) || models.get(obj.kind);
+};
+
+export const createRequests = (objs, models, isDryRun) => {
```

Comment:
nit: The name here is a bit confusing since this actually makes the HTTP requests. It kind of sounds like it's creating the request body, but not firing the request. Maybe

```suggestion
export const createResources = (objs, models, isDryRun) => {
```